src/wallet/components/wallet-advanced-send/components/SendAddressSelector.tsx

```diff
@@ -0,0 +1,59 @@
+'use client';
+
+import { Address, Avatar, Name } from '@/identity';
+import { background, border, cn, pressable } from '@/styles/theme';
+import type { SendAddressSelectorProps } from '../types';
+
+export function SendAddressSelector({
+  address,
+  senderChain,
+  handleClick,
```

Comment:
suggestion: IMO, the `handleEvent` convention makes sense for the callback function name, but the prop name should be `onEvent`. That's more in line with regular DOM elements (e.g. `<button onClick={...} />`)

Reply:
sounds good, updated